test/fixtures/browser-app/mixins/time-stamps.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(Model, options) {
+
+  Model.timeStampsMixin = true;
```

Comment:
AFAICT, the browser test is not checking whether `Model.timeStampsMixin` was set. Please modify the browser fixture app to use this mixin and then verify in the unit-test that the flag was set.

I think this is what you need to do:
 1. Modify the file `test/fixtures/browser-app/models/customer.json` and add `"mixins": {"TimeStamps": {} }`
 2. In the test, check that `app.models.Customer.timeStampsMixin` is `true`.